common/ratelimit/limiter.go

```diff
@@ -129,10 +129,15 @@ func (d *rateLimiter) checkAllowed(ctx context.Context, params common.RequestPar
 				"requester_id":   params.RequesterID,
 				"requester_name": params.RequesterName,
 				"bucket_index":   strconv.Itoa(i),
-			}).Set(float64(bucketParams.BucketLevels[i]))
+			}).Set(float64(bucketLevels[i]))
```

Comment:
not understanding it. Wouldn't the bucketLevel get updated anyway in both cases?